### PR TITLE
Add project reorder docs

### PR DESCRIPTION
**Pull request by thecraftman**
- add documentation for project re-order

### CODE REVIEW

File: apl/tabular-operators/project-reorder-operator.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+```kusto
+| project-reorder FieldName1, FieldName2, ... [asc|desc]
+```
+
+## Arguments
+
+| **name**              | **type**     | **description**                                      |
+| --------------------- | ------------ | ------------------------------------------------------ |
+| Field Name            | **string**     | The name of the field to be reordered in the output. |
+| [asc|desc]           | **string** |  Optional. Specifies the sort order for the reordered fields. |
```
Comment (ted-gould) [inline apl/tabular-operators/project-reorder-operator.mdx]: We support the `granny-asc` and `granny-desc` as well

File: apl/tabular-operators/project-reorder-operator.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+tags: 
+    ['axiom documentation', 'documentation', 'axiom', 'tabular operators', 'project-reorder']
+---
+
+Reorders specified fields in the output while keeping the original order of unspecified fields.
+
+## Syntax
+
+```kusto
+| project-reorder FieldName1, FieldName2, ... [asc|desc]
```
Comment (ted-gould) [inline apl/tabular-operators/project-reorder-operator.mdx]: This should be: `project-reorder FieldName1OrWildcard [asc|desc], FieldName2OrWildcard [direction], ...`

**Comment by ted-gould** [comment]
image

It doesn't seem to be in the TOC.

File: apl/tabular-operators/project-reorder-operator.mdx (hunk 1 above)
Comment (gordallott) [reply apl/tabular-operators/project-reorder-operator.mdx]: just for some extra description on grannies 
> granny-asc or granny-desc orders by ascending or descending, respectively, while secondarily sorting by the next numeric value. For example, a100 comes before a20 when granny-asc is specified.

that's from the kql docs

File: apl/tabular-operators/project-reorder-operator.mdx (hunk 2 above)
Comment (gordallott) [reply apl/tabular-operators/project-reorder-operator.mdx]: we support wildcard suffix matches so probably more 

> `project-reorder FieldName[*] [asc|desc|granny-asc|granny-desc], ...`

both of these are valid 
```kql
T
| project-reorder testing* desc // orders all the fields starting with testing first, then all the other fields
``` 

```kql
T
| project-reorder trace_id, *, span_id // orders the trace_id then everything else, then span_id fields 
```

**Review by gordallott — CHANGES_REQUESTED** [review]

File: apl/tabular-operators/project-reorder-operator.mdx (hunk 1 above)
Comment (thecraftman) [reply apl/tabular-operators/project-reorder-operator.mdx]: @gordallott makes sense, added. all good now

**Review by gordallott — APPROVED** [review]
:+1:

**Pull request merged**